import/bemobil_xdf2bids.m

```diff
@@ -417,124 +429,185 @@ function bemobil_xdf2bids(bemobil_config, numericalIDs, varargin)
-                        
+
                         %--------------------------------------------------
                         if ~exist('motionInfo', 'var')
-                            
+
                             % data type and acquisition label
                             motionInfo.acq                                     = 'Motion';
-                            
+
                             % motion specific fields in json
                             motionInfo.motion.Manufacturer                     = 'Undefined';
                             motionInfo.motion.ManufacturersModelName           = 'Undefined';
                             motionInfo.motion.RecordingType                    = 'continuous';
-                            
+
                             % coordinate system
                             motionInfo.coordsystem.MotionCoordinateSystem      = 'Undefined';
                             motionInfo.coordsystem.MotionRotationRule          = 'Undefined';
                             motionInfo.coordsystem.MotionRotationOrder         = 'Undefined';
-                            
+
                         end
+
+                        % sampling frequency
+                        motionInfo.motion.TrackingSystems.(motionInfo.motion.tracksys{si}).SamplingFrequencyEffective = motion.hdr.Fs;
+                        
+                        if strcmpi(motionInfo.motion.TrackingSystems.(motionInfo.motion.tracksys{si}).SamplingFrequencyNominal, 'n/a')
+                           motionInfo.motion.TrackingSystems.(motionInfo.motion.tracksys{si}).SamplingFrequencyNominal = motion.hdr.nFs;
+                        end 
+                        
+                        
+                          
                         
                         % data type and acquisition label
                         motioncfg.acq                                     = motionInfo.acq;
-                        
+
                         % motion specific fields in json
                         motioncfg.motion                                  = motionInfo.motion;
                         
+                        % tracking system
+                        motioncfg.tracksys                                = motionInfo.motion.tracksys {si}; % has to be adjusted for multiple tracking systems in one session
+                        motioncfg.motion.tracksys_all                     = motionInfo.motion.tracksys ; % needed for removing general trackingsys info 
+            
+                        % number of all tracking systems used in session
+                        motioncfg.motion.TrackingSystemCount              = numel(xdfmotion);
+
                         % start time
-                        motioncfg.motion.StartTime                        = motionStartTime - eegStartTime;
-                        
+                        motioncfg.motion.start_time                       = motionStartTime - eegStartTime;
```

Comment:
What is the idea behind changing from StartTime to start_time? This input is passed to data2bids, to be entered in the .json file. There field names are assumed to be camel case.